code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -170,7 +170,7 @@
 	for(var/datum/powernet/PN in powernets)
 		for(var/obj/structure/cable/C in PN.cables)
 			var/turf/simulated/floor/F = get_turf(C)
-			if(istype(F,/turf/simulated/floor) && !F.floor_tile)
+			if(istype(F,/turf/simulated/floor) && !F.floor_tile && C.z == 1)
```

Comment:
STATION_Z

Reply:
there goes the pulse demon on the derelict gimmick

Reply:
map.zMainStation is actually better and more flexible, so I used that, but thanks for pointing it out.